Uml.Robotics.Ros/_Init.cs

```diff
@@ -325,7 +338,15 @@ public static void Init(IDictionary<string, string> remappingArgs, string name,
                         msgRegistry.ParseAssemblyAndRegisterRosMessages(assembly);
                         srvRegistry.ParseAssemblyAndRegisterRosServices(assembly);
                     }
-
+#else
+                    // Load RosMessages from Messages assembly
+                    var msgAssembly = Assembly.LoadFrom("Uml.Robotics.Ros.dll");
+
+                    logger.LogDebug($"Parse assembly: {msgAssembly.Location}");
+                    msgRegistry.ParseAssemblyAndRegisterRosMessages(msgAssembly);
+                    srvRegistry.ParseAssemblyAndRegisterRosServices(msgAssembly);
+             
```

Comment:
This does not yet look complete to me. The system should scan for Assemblies that  reference the Uml.Robotics.Ros.MessageBase assembly and then register all messages/service type serializers/deserializers of those.

Reply:
not being able to use the "System.Runtime.Loader" library, I had to fall back for this solution, I thought it could be fine.
Do you have a suggestion on how to improve this part?

Reply:
not being able to use the "System.Runtime.Loader" library, I had to fall back for this solution, I thought it could be fine.
Do you have a suggestion on how to improve this part?

Reply:
What I meant is: The process should load all the ROS Message Assemblies (e.g. that have been identified by `MessageTypeRegistry.GetCandidateAssemblie()`).